packages/rule-components/src/RuleTable/RuleTable.js

```diff
@@ -159,7 +165,7 @@ class RuleTable extends Component {
             />
             <Table
                 {...props}
-                {...sortBy && { onSort: this.onSort }}
+                {...sortProps}
```

Comment:
No need to assign to a constant
```JSX
<Table
  { ...sortBy && { ...sortBy, onSort: this.onSort } }
```

Reply:
I think it's better to leave it as it is, because otherwise it's hard to understand.

Reply:
That's not hard to understand, it is used on many other places and this way we are not using constant that is used on one place.

Reply:
Well, I’m not sure about that, because when I tried to find what was broken, I find out that `onSort` just didn’t insert into RuleTable props, so it was not so obvious construction.

Reply:
anybody from @RedHatInsights/ui-reviewers any thoughts on this?

Reply:
I am understanding what bond95 means but Karel's construction is really simple - the difference in readability is really small in this case, so I would go with that as it is a suggestion from a maintainer, who will take over this code.

>  I find out that onSort just didn’t insert into RuleTable props,

Actually I would say this would be faster to find in the Karel's solution. Using the separate constant, you have to go up and look, what it is. Otherwise the code is basically the same.

Reply:
@rvsia thank you for your response. Ok, 2 vs 1, so I'll return it as it was.

Reply:
@karelhala @rvsia done